packages/charts/README.md

```diff
@@ -0,0 +1,7 @@
+# Shoreline Charts
+
+`shoreline-components` and `echarts` are peer dependencies of `shoreline-charts`
```

Comment:
Could you add a description about the charts, beside how rot run, test and keep it?